expo-maps/ios/ExpoMaps/PropsData.swift

```diff
@@ -27,20 +27,20 @@ struct Point: Record {
 
 struct PolygonObject: Record {
   @Field var points: [Point] = []
-  @Field var fillColor: UIColor?
-  @Field var strokeColor: UIColor?
-  @Field var strokeWidth: Float?
+  @Field var fillColor: UIColor? = nil
+  @Field var strokeColor: UIColor? = nil
+  @Field var strokeWidth: Float? = nil
```

Comment:
@p-karcz Isn't it defaulted to nil?

Reply:
Without defining default values we have to use constructor with fields wrapped inside `Field` class